py3status/modules/velib_metropole.py

```diff
@@ -0,0 +1,292 @@
+# -*- coding: utf-8 -*-
+"""
+Display Velib shared bike avaibility on our favorite stations.
+Use https://www.velib-metropole.fr/map data.
+
+You only need to set id of stations you want to monitor in station_codes param.
+Then scroll to cycle stations, and middle clic to force a refresh.
+
+Configuration parameters:
+    button_next: Display next station (default 5)
+    button_previous: Display previous station (default 4)
+    button_refresh: refresh stations (default 2)
```

Comment:
ok i hide refresh button

Reply:
>  I think it'd be good if we start hiding this option everywhere. 

why?  this is the most useful option

Reply:
Four modules have this option right now. 
```python
./whatismyip.py:175:        elif button == self.button_refresh:
./github.py:259:        elif button != self.button_refresh:
./gitlab.py:165:        if button != self.button_refresh:
./google_calendar.py:496:            elif button == self.button_refresh:
```
Option 1) We can add this option for API modules everywhere. 
Option 2) We can `getattr` those 4 modules into nonexistent.

If we go with Option 2, then users can refresh modules everywhere with button `2`. This is already enforced for long time in many modules thanks to your code. Users can pick up on this soon enough or they will learn about this with (new) documentation. 

I think this could be akin to `request_timeout` in that it's good to expose those options, but maybe users won't care (and that it will refresh again in a minute) and maybe we should try and hide this config to simplify code and/or to enforce consistency refresh behavior for all modules.

Reply:
@lasers The thing is not to change __anything__ but to create an issue to discuss a change before implementing

Also just to let you know I'll be merging #1504 soon as I have sign-off on it - I'll look at fixing lm_sensors too

Reply:
lovers, so, I hide / hardcode refresh or not? 🐌